docs/framework/20categories/10network/network.md

```diff
@@ -1,2 +1,18 @@
 ## Network Consensus Risk
-TODO: The EEA Security Guidelines has some good words for this section.
+Cross-chain protocols enable the coordination of state changes across networks. This typically requires sending state information from a source network to one or more destination networks and performing consequent state changes. A fundamental assumption in this process is that: a) states in the source network are valid and final according to its consensus rules, and b) the underlying network can process all transactions in a timely manner.
+
+From the perspective of risk in cross-chain protocols, there are at least three considerations relating to these assumptions:
+1. Protocols should only communicate states that have been finalized in the source network
+1. Protocol security should be resilient to network liveness failures
+1. Protocols should mitigate the impact of network safety violations
+
+#### Transaction Finality
+Finality models vary significantly across networks. They could be _probabilistic_ or _deterministic_, _instant_ or _eventual_. Networks with _deterministic_ finality offer absolute assurance about the state of a network after some time. In contrast, probabilistic models offer only degrees of confidence, with more crypto-economic assurance achieved over time. For _deterministic_ models, finality could be instantly achieved after a single block or eventually after several blocks. Networks with slower finality times increase latency in cross-chain communication, which impacts the customer experience. However, cross-chain protocols must ensure that the finality of a given state is achieved before relaying the state across chains. Cross-chain protocols, which optimize for performance and relay information across chains without sufficiently waiting for finality, risk creating inconsistencies across networks and the possible loss of user funds.
```

Comment:
The term finality should be introduced. Something like, 
"A transaction is deemed to be final when the block that it is contained in is final. "

Reply:
For probabilistic models, is it "crypto-economic assurance"? For Bitcoin, for instance, the result is economic, but it comes down to having enough hashing power to find a forked chain with greater difficulty.

Reply:
I think that instant finality should be called out as a special case of deterministic finality. I don't know that eventual finality should be defined, as it seems to just be deterministic finality. 
The term eventual also has the connotation of "eventual consistent". This term has been mistakenly used in relation to finality for PoW networks, and hence I think should be avoided.

Reply:
"risk creating inconsistencies across networks and the possible loss of user funds", if a Liquidity Provider fronts funds by giving them to a user on a destination chain, then it is the LP that loses the funds. In that situation, the user may end up keeping their funds on the source chain and have the funds on the destination chain.

Reply:
> The term finality should be introduced. Something like, "A transaction is deemed to be final when the block that it is contained in is final. "

Introducing the term makes sense

> For probabilistic models, is it "crypto-economic assurance"? For Bitcoin, for instance, the result is economic, but it comes down to having enough hashing power to find a forked chain with greater difficulty.

Yea, that's true, but I'd argue there is an economic component to the assurance here, both in terms of expended costs and forgone rewards that one can infer about a competing fork. Given it does not add much to the discussion, though, I've removed the word.

> I think that instant finality should be called out as a special case of deterministic finality. 

I've rephrased a few sentences that I believe would address your comments and offer more clarity

> if a Liquidity Provider fronts funds by giving them to a user on a destination chain, then it is the LP that loses the funds. 

right, I was referring to loss of funds by either party, instead of just the user. Reworded for clarity.